### PR TITLE
iBBQ first commit

### PR DESCRIPTION
**Pull request by jedgarpark**

### CODE REVIEW

**Comment by ladyada** [comment]
@dhalbert please review :)

File: CLUE_BBQ/clue_bbq.py [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+                    (5, "", PURPLE, 135, 200)
+                   ]
+
+my_labels = {}  # dictionary of configured my_labels
+
+text_group = displayio.Group(max_size=8, scale=1)
+
+for label_config in my_labels_config:
+    (name, text, color, x, y) = label_config  # unpack a tuple into five var names
+    # old way sould have been: name = label_config[0], text = label_config[1]
```
Comment (dhalbert) [inline CLUE_BBQ/clue_bbq.py:73]: typo in comment ("sould"), and maybe this comment should just be removed?

File: CLUE_BBQ/clue_bbq.py [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+                if temps is not None:
+                    probe_count = len(temps)  # check how many probes there are
+                    for i in range(probe_count):
+                        if (temps[i] is not 0 and temps[i] < 1000):  # unplugged probes
+                            if unit_mode:
+                                clue.pixel.fill((50, 0, 0))
+                                temp = temps[i]
+                                my_labels[i].text = "{} C".format(temp)
+                                clue.pixel.fill((0, 0, 0))
+                                print("Probe", i+1, "Temperature:", temp,"C",)
```
Comment (dhalbert) [inline CLUE_BBQ/clue_bbq.py:134]: Run `black` over this code to reformat it to specs (e.g. missing space here, extra comma)

File: CLUE_BBQ/clue_bbq.py [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+                    clue.play_tone(1200, 0.1)
+                    clue.red_led = False
+                    time.sleep(0.1)  # debounce
+
+                temps = ibbq_service.temperatures
+                batt = ibbq_service.battery_level
+                if temps is not None:
+                    probe_count = len(temps)  # check how many probes there are
+                    for i in range(probe_count):
+                        if (temps[i] is not 0 and temps[i] < 1000):  # unplugged probes
```
Comment (dhalbert) [inline CLUE_BBQ/clue_bbq.py:128]: Don't use `is not` for numerical comparisons, use `!=`. `is not` should be used only to see if two objects are not the same object (e.g. `is not None`).

File: CLUE_BBQ/clue_bbq.py [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+                                my_labels[i].text = "{} F".format(temp)
+                                clue.pixel.fill((0, 0, 0))
+                                print("Probe", i+1, "Temperature:", temp,"F",)
+                        else:
+                            print("Probe", i+1, "is unplugged",)
+                            my_labels[i].text = ("  ---")
+                    clue.display.show(temperatures_screen)
+
+    except _bleio.ConnectionError:
+        # Redisplay splash screen here, or put it at top of while True:
```
Comment (dhalbert) [inline CLUE_BBQ/clue_bbq.py:147]: Remove this comment; it was just a note to you in the sample code I sent you.

File: CLUE_BBQ/clue_bbq.py (hunk 1 above)
Comment (jedgarpark) [reply CLUE_BBQ/clue_bbq.py:73]: removing it

File: CLUE_BBQ/clue_bbq.py (hunk 3 above)
Comment (jedgarpark) [reply CLUE_BBQ/clue_bbq.py:128]: changed it

File: CLUE_BBQ/clue_bbq.py (hunk 4 above)
Comment (jedgarpark) [reply CLUE_BBQ/clue_bbq.py:147]: gone

File: CLUE_BBQ/clue_bbq.py (hunk 2 above)
Comment (jedgarpark) [reply CLUE_BBQ/clue_bbq.py:134]: done

File: CLUE_BBQ/clue_bbq.py [hunk 5]
```diff
@@ -122,27 +123,28 @@
 
                 temps = ibbq_service.temperatures
                 batt = ibbq_service.battery_level
-                if temps is not None:
+                if temps != None:
                     probe_count = len(temps)  # check how many probes there are
                     for i in range(probe_count):
-                        if (temps[i] is not 0 and temps[i] < 1000):  # unplugged probes
+                        if temps[i] is not 0 and temps[i] < 1000:  # unplugged probes
```
Comment (dhalbert) [inline CLUE_BBQ/clue_bbq.py]: Sorry if I wasn't clear. It should be `temps is not None` on line 126. and `temps[i] != 0` on line 129.

Comment (jedgarpark) [reply CLUE_BBQ/clue_bbq.py]: ah, OK, got it.

**Pull request merged**